be1-go/channel/generalChirping/mod.go

```diff
@@ -195,38 +190,38 @@ func (c *Channel) VerifyBroadcastMessage(broadcast method.Broadcast) error {
 	return nil
 }
 
-// AddChirp checks and stores an add chirp message
-func (c *Channel) addChirp(msg message.Message) error {
-	err := c.verifyChirpBroadcastMessage(msg)
+// addChirp checks and stores an add chirp message
+func (c *Channel) addChirp(msg message.Message, msgData interface{}) error {
+	data, ok := msgData.(*messagedata.ChirpNotifyAdd)
+	if !ok {
+		return xerrors.Errorf("message %v isn't a chirp#notifyAdd message", msgData)
+	}
+
+	err := c.verifyNotifyChirp(msg, data)
 	if err != nil {
 		return xerrors.Errorf("failed to get and verify add chirp message: %v", err)
 	}
-	c.inbox.StoreMessage(msg)
 
 	return nil
 }
 
-// DeleteChirp checks and stores a delete chirp message
-func (c *Channel) deleteChirp(msg message.Message) error {
-	err := c.verifyChirpBroadcastMessage(msg)
+// deleteChirp checks and stores a delete chirp message
```

Comment:
```suggestion
// deleteChirp checks a delete chirp message
```

Reply:
More like I hate myself for not seeing it myself :)